calico_containers/tests/unit/calicoctl_test.py

```diff
@@ -25,11 +29,12 @@
 from calico_ctl.node import validate_arguments as node_validate_arguments
 from calico_ctl.pool import validate_arguments as pool_validate_arguments
 from calico_ctl.profile import validate_arguments as profile_validate_arguments
```

Comment:
Not planning to change as we should address this as part of the split-out into separate files.  Once split out into individual files we can just import validate_arguments without conflict.